xmtp_mls/src/groups/mod.rs

```diff
@@ -387,38 +377,36 @@ where
         }
     }
 
-    pub fn process_messages(&self, envelopes: Vec<Envelope>) -> Result<(), GroupError> {
+    pub(crate) fn process_messages(&self, envelopes: Vec<Envelope>) -> Result<(), GroupError> {
         let mut conn = self.client.store.conn()?;
         let provider = self.client.mls_provider(&mut conn);
         let mut openmls_group = self.load_mls_group(&provider)?;
```

Comment:
Could merge this with the two lines above now that we don't need access to the provider or conn below this point

Reply:
I'm not sure what you mean by merging - like put it all in the same line? I think that just makes it less readable :/

Reply:
It is less readable, but it does return the connection to the pool sooner. 

Not a big deal one way or the other.